config/action.d/shorewall-ipset-proto6.conf

```diff
@@ -0,0 +1,84 @@
+# Fail2Ban configuration file
+#
+# Author: Eduardo Diaz
+#
+# This is for ipset protocol 6 (and hopefully later) (ipset v6.14).
+# for shorewall
+#
+# Use this setting in jail.conf to modify the name and the max time in every jain
+# action   = shorewall-ipset-proto6[name=SSH, bantime=10000]
+#
+# This requires the program ipset which is normally in package called ipset.
+#
+# IPset was a feature introduced in the linux kernel 2.6.39 and 3.0.0 kernels.
+#
+# The default Shorewall configuration is with "BLACKLISTNEWONLY=Yes" (see
+# file /etc/shorewall/shorewall.conf). This means that when Fail2ban adds a
+# new shorewall rule to ban an IP address, that rule will affect only new
+# connections. So if the attempter goes on trying using the same connection
+# he could even log in. In order to get the same behavior of the iptable
+# action (so that the ban is immediate) the /etc/shorewall/shorewall.conf
+# file should me modified with "BLACKLISTNEWONLY=No".
+#
+# A lot of this depends of the shorewall version at least have the 4.5.5 Version
```

Comment:
BLACKLISTNEWONLY is very util, if are yes only ban the new connexions if are in no ban all.

This is importan con avoid for example some DDOS if you bann all the conexions the conexion open to ssh for example will keep open until the system closes the conexion in a big DOS you can keep a big about conexion to close and the system can have problems open more process..

Some time is better only blacklist new only.